esp-alloc/Cargo.toml

```
@@ -23,10 +23,24 @@ default-target = "riscv32imc-unknown-none-elf"
 features       = ["nightly"]
 
 [dependencies]
+defmt = { version = "0.3.8", optional = true }
+cfg-if                = "1.0.0"
 critical-section      = "1.1.3"
 enumset               = "1.1.5"
 linked_list_allocator = { version = "0.10.5", default-features = false, features = ["const_mut_refs"] }
+document-features = "0.2.10"
```

Just a nitpick, but could you please align the indentation 😅 
```suggestion
defmt                 = { version = "0.3.8", optional = true }
cfg-if                = "1.0.0"
critical-section      = "1.1.3"
enumset               = "1.1.5"
linked_list_allocator = { version = "0.10.5", default-features = false, features = ["const_mut_refs"] }
document-features     = "0.2.10"
```